templates/content/social-media-links.html.twig

```diff
@@ -1,6 +1,9 @@
 <div class="social-media__items">
   {% set classes = ['social-media__item'] %}
-  {% for element in elements %}
+  {% for key, element in elements %}
+    {# Only email text is translated #}
+    {% set use_lang_attributes = alternative_language and key != 'email' %}
```

Comment:
I wonder if we should change the email contexts somehow as we can not be 100% sure to which languages email has been translated of all the world languages. Like for example, is it translate to Sami?

Reply:
Ah maybe it's the "Send as an email" text. Some languages don't have translation for it and then the "lang" and "dir" attributes would be needed.

Reply:
This is now fixed. If "Send as an email" text is not translated, a `lang `and `dir` attributes are added.